css/index.css

```diff
@@ -49,13 +48,37 @@ table {
 
 * {
     box-sizing: border-box;
+    max-width: 100%;
 }
 
 html, body {
     height: 100%;
     font-family: 'Roboto', sans-serif;
 }
 
+header {
+    display: flex;
+    justify-content: space-evenly;
+}
+
+nav { 
+    display: flex;
+    justify-content: center;
+    padding: 1%;
+    width: 32.2%;
+    justify-content: space-between;
+
+}
+nav a {
+    text-decoration: none;
+    color: black;
+    font-weight: 600;
+}
+.Navigation {
```

Comment:
There is another flexbox justify-content that will justify the content like it is seen in the design file.  Space evenly puts the same amount of space between each anchor and this is okay.  Space evenly will space the anchors out as well but will also align them to the beginning and end of the div.